json/src/lib.rs

```diff
@@ -292,6 +292,15 @@ pub struct SignRawTransactionResult {
     pub errors: Vec<SignRawTransactionResultError>,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
This line is not necessary as the only field which not a single word has a `serde::rename` on it.

Reply:
I agree